yrs/src/undo.rs

```diff
@@ -35,7 +35,6 @@ use crate::{DeleteSet, Doc, Observer, Transact, TransactionMut, ID};
 /// - [UndoManager::observe_item_popped], which is fired whenever [StackItem] is being from undo
 ///    manager as a result of calling either [UndoManager::undo] or [UndoManager::redo] method.
 #[repr(transparent)]
-#[derive(Clone)]
```

Comment:
Actually, relying on derived clone method here for Undo will break all `UndoManager` methods using `&mut self`.